packages/event-handler/src/appsync-events/AppSyncEventsResolver.ts

```diff
@@ -87,11 +94,12 @@ class AppSyncEventsResolver extends Router {
     }
 
     if (isAppSyncEventsPublishEvent(event)) {
-      return await this.handleOnPublish(event, context);
+      return await this.handleOnPublish(event, context, options);
```

Comment:
Is there a reason why we're calling `await` in the return statement? Is it because of this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await#improving_stack_trace?